src/core/api/io_lollipop/v1/_openapi.yaml

```diff
@@ -0,0 +1,161 @@
+openapi: 3.0.1
+info:
+  version: 1.0.0
+  title: IO Lollipop Function Lollipop Consumer API
+  x-logo:
+    url: https://io.italia.it/assets/img/io-logo-blue.svg
+  description: >
+    Documentation of the IO Lollipop Function API exposed to Lollipop
+    Consumerhere.
+servers:
+  - url: https://api.pagopa.it/api/v1/lollipop
+security:
```

Comment:
The security definition on swagger say that:
A declaration of which security mechanisms can be used across the API. The list of values includes alternative security requirement objects that can be used. Only one of the security requirement objects need to be satisfied to authorize a request. 

In our case, we want both the security mechanism

Reply:
This is the exact openapi spec we have on fn-lollipop. I think this is why this is still in draft. When those definitions will be ready, they will be imported here. So we should put this comment on the other PR.